src/components/alert/alert.css

```diff
@@ -0,0 +1,6 @@
+.fade-enter-active, .fade-leave-active {
+    transition: opacity .15s linear;
+}
+.fade-enter, .fade-leave-to {
+    opacity: 0;
+}
```

Comment:
I wonder if we should target these classes to elements containing the class `.alert`, so that it doesn't interfere with users who may have their own fade transition with different settings?

Maybe:

```css
.alert.fade-enter-active,
.alert.fade-leave-active {
     transition: opacity .15s linear;
 }
.alert.fade-enter,
.alert.fade-leave-to {
     opacity: 0;
 }
```